tq42/utils/utils.py

```diff
@@ -103,3 +106,26 @@ def dynamic_create_exp_run_request(
         **keyword_args,
     )
     return experiment_request
+
+
+def save_token(service_name: str, backup_save_path: str, token: str) -> None:
```

Comment:
Maybe use the return value for returning which location was used whether the path or the keyring.